platforms/emulator/rom/src/start.s

```diff
@@ -26,6 +26,15 @@ _start:
     # Initialize the stack pointer
     la sp, STACK_TOP
 
+    # Initialize MRAC (Region Access Control Register)
+    # MRAC controls cacheability and side effects for 16 memory regions (256MB each)
+    # The value is computed from the memory map at build time
+    # CSR address 0x7c0 = MRAC register
+    # Use lui/addi to load the 32-bit constant properly
+    lui     t0, %hi(MRAC_VALUE)
```

Comment:
i think LA does the same thing but wasnt sure. Can replace with LA if equivalent.

Reply:
Either is fine. I usually use `li` or `la` because I never remember the `%hi` syntax :)